tests/conftest.py

```diff
@@ -8,9 +8,26 @@
 import cPickle
 import re
 
+import py
 import webob
 import pytest
 from webob.request import Request
+from beaker.middleware import SessionMiddleware
+
+@pytest.fixture(scope="module")
+def session_dir(request):
+    '''For testing we should manage the session directory ourselves so that:
+       a) we don't stomp on existing session directories
+       b) we don't require run time configuration
+       c) we ensure that everything gets properly cleaned up after the test run
+    '''
+    dirname = py.path.local(mkdtemp())
+    request.addfinalizer(lambda: dirname.remove(rec=1, ignore_errors=True))
+    return str(dirname)
+
+@pytest.fixture(scope="module")
+def static_url_space():
```

Comment:
This is not used and frontend is not defined in this module. What was the plan for this?